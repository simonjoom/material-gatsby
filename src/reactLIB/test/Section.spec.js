import React from 'react';
import { shallow } from 'enzyme';
import Section from '../src/Section';

describe('<Section />', () => {
  let wrapper;

  test('renders', () => {
    wrapper = shallow(<Section />);
    expect(wrapper).toMatchSnapshot();
  });

  test('renders children', () => {
    wrapper = shallow(
      <Section>
        <p>nope</p>
      </Section>
    );
    expect(wrapper).toMatchSnapshot();
  });
});
