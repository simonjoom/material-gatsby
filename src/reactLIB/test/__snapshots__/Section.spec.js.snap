// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Section /> renders 1`] = `
<div
  className="section"
/>
`;

exports[`<Section /> renders children 1`] = `
<div
  className="section"
>
  <p>
    nope
  </p>
</div>
`;
